yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -172,58 +204,155 @@ func backupLogFilesFn() {
 	if !backupLogFiles {
 		return
 	}
-	utils.PrintAndLog("backing up log files")
 	// TODO: in case of failures when cmd is executed again, even if logs were backed up, new log file for end migration will come up
-	err := os.Rename(filepath.Join(exportDir, "logs"), filepath.Join(backupDir, "logs"))
+
+	backupLogDir := filepath.Join(backupDir, "logs")
+	err := os.MkdirAll(backupLogDir, 0755)
+	if err != nil {
+		utils.ErrExit("end migration: creating logs directory for backup: %v", err)
+	}
+
+	utils.PrintAndLog("backing up log files")
+	files, err := os.ReadDir(filepath.Join(exportDir, "logs"))
 	if err != nil {
-		utils.ErrExit("end migration: moving log files: %v", err)
+		utils.ErrExit("end migration: reading logs directory: %v", err)
 	}
+
+	for _, file := range files {
+		if file.IsDir() || !strings.HasSuffix(file.Name(), ".log") {
+			continue
+		}
+
+		logFilePath := filepath.Join(exportDir, "logs", file.Name())
+		backupFilePath := filepath.Join(backupDir, "logs", file.Name())
+		err = os.Rename(logFilePath, backupFilePath)
+		if err != nil {
+			utils.ErrExit("end migration: moving log files: %v", err)
+		}
+	}
+}
+
+func askPassword(destination string, user string, envVar string) (string, error) {
+	if os.Getenv(envVar) != "" {
+		return os.Getenv(envVar), nil
+	}
+
+	if user == "" {
+		fmt.Printf("Password to connect to %s (In addition, you can also set the password using the environment variable '%s'): ",
+			destination, envVar)
+	} else {
+		fmt.Printf("Password to connect to '%s' user of %s (In addition, you can also set the password using the environment variable '%s'): ",
+			user, destination, envVar)
+	}
+	bytePassword, err := term.ReadPassword(int(syscall.Stdin))
+	if err != nil {
+		return "", fmt.Errorf("reading password: %w", err)
+	}
+	fmt.Print("\n")
+	return string(bytePassword), nil
 }
 
 func cleanupSourceDB(msr *metadb.MigrationStatusRecord) {
-	utils.PrintAndLog("cleaning up source db...")
+	utils.PrintAndLog("cleaning up voyager state from source db...")
 	source := msr.SourceDBConf
 	if source == nil {
 		log.Info("source db conf is not set. skipping cleanup")
 		return
 	}
-	err := source.DB().Connect()
+
+	var err error
+	source.Password = sourceDBPassword
+	if sourceDBPassword == "" {
```

Comment:
What if source is oracle and using tns-alias for connection?

Reply:
sourceDBPassword will still be required, right? TNSAlias doesn't have the username/password